### PR TITLE
Feature/playlist ui

### PR DESCRIPTION
**Pull request by KlaraMinsterova**
Basic UI for playlists created

### CODE REVIEW

File: web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    @RequestMapping(value = "/detail/{id}", method = RequestMethod.GET)
+    public String view(@PathVariable("id") Long id, Model model) {
+        log.debug("detail({})", id);
+        model.addAttribute("playlist", playlistFacade.findPlaylistById(id));
+        return "playlist/detailPlaylist";
+    }
+
+    @RequestMapping(value = "/new", method = RequestMethod.GET)
+    public String newProduct(Model model) {
```
Comment (xziska02) [inline web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java]: newPlaylist

File: web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        PlaylistDto playlistDto = new PlaylistDto();
+        playlistDto.setDateCreated(new Date());
+        //TODO get logged user
```
Comment (xziska02) [inline web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java]: TODO -> get inspiration of getting logged user as I did in UserController

File: web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        model.addAttribute("playlist", playlistDto);
+        return "playlist/newPlaylist";
+    }
+
+    @RequestMapping(value = "/create", method = RequestMethod.POST)
+    public String create(@Valid @ModelAttribute("playlist") PlaylistDto formBean, BindingResult bindingResult,
+                         Model model, RedirectAttributes redirectAttributes) {
```
Comment (xziska02) [inline web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java]: not used redirectAttributes -> remove then

File: web/src/main/webapp/WEB-INF/jsp/playlist/allPlaylists.jsp [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            </thead>
+            <tbody>
+                <c:forEach items="${playlists}" var="playlist">
+                    <tr>
+                        <td class="col-md-3"><c:out value="${playlist.title}"/></td>
+                        <td class="col-md-3"><c:out value="${playlist.dateCreated}"/></td>
+                        <td class="col-md-3"><c:out value="${playlist.user.username}"/></td>
+                        <td class="button">
+                            <form method="get"
+                                  action="/pa165/playlist/detail/${playlist.playlistId}" >
```
Comment (xziska02) [inline web/src/main/webapp/WEB-INF/jsp/playlist/allPlaylists.jsp]: wrong redirect -> without pa165

Comment (KlaraMinsterova) [reply web/src/main/webapp/WEB-INF/jsp/playlist/allPlaylists.jsp]: doesn't work to me without that

File: web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java [hunk 5]
```diff
@@ -70,8 +79,23 @@ public String create(@Valid @ModelAttribute("playlist") PlaylistDto formBean, Bi
             return "playlist/newPlaylist";
         }
 
+        redirectAttributes.addFlashAttribute("alert_success", "Playlist was created.");
+
         playlistFacade.createPlaylist(formBean);
```
Comment (xziska02) [inline web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java]: you need to make connection between user and playlist -> update user in DB | userFacade.updateUser(userDto);
and before update you should add playlist into userDto

Comment (KlaraMinsterova) [reply web/src/main/java/cz/muni/fi/pa165/music_library/controllers/PlaylistController.java]: no, I don't, hibernate takes care of that

you can check it quite easily, if you create new playlist and go to user profile or song adding where users playlists are listed, you can see it's already there

**Pull request merged**